tests/run-make/autodiff/type-trees/mixed-struct-typetree/mixed.check

```diff
@@ -0,0 +1,2 @@
+; Check that mixed struct with large array compiles successfully
+CHECK: define{{.*}}"enzyme_type"="{[]:Float@float}"{{.*}}@test_mixed_struct{{.*}}"enzyme_type"="{[]:Pointer}"
```

Comment:
This test also doesn't show anything, right now almost every test has just `enzyme_type"="{[]:Pointer}`, you'd want different tests with different inputs to generate different enzyme_type's. The interesting part is only after the pointer.